ansible/roles/kong-api/defaults/main.yml

```diff
@@ -1291,319 +1092,71 @@ kong_apis:
     - "{{ statsd_pulgin }}"
     - name: acl
       config.whitelist:
-        - 'appAccess'
-    - name: rate-limiting
-      config.policy: local
-      config.hour: "{{ medium_rate_limit_per_hour }}"
-      config.limit_by: credential
-    - name: request-size-limiting
-      config.allowed_payload_size: "{{ small_request_size_limit }}"
-
-  - name: flagContent
-    uris: "{{ content_prefix }}/v1/flag"
-    upstream_url: "{{ knowledge_mw_service_url }}/v1/content/flag"
-    strip_uri: true
-    plugins:
-    - name: jwt
-    - name: cors
-    - "{{ statsd_pulgin }}"
-    - name: acl
-      config.whitelist:
-        - 'contentAccess'
-    - name: rate-limiting
-      config.policy: local
-      config.hour: "{{ medium_rate_limit_per_hour }}"
-      config.limit_by: credential
-    - name: request-size-limiting
-      config.allowed_payload_size: "{{ small_request_size_limit }}"
-
-  - name: freshDesk
```

Comment:
This is a Desktop portal API do we need to remove this @amit-tarento  @Hari-stackroute ?

Reply:
@SMYALTAMASH This API is not removed, its  there in a diff line

Reply:
These APIs are still there , but git shows got changed , you can search this API still exist.
Might be because of lot of deletion